pkg/infra/error.go

```diff
@@ -0,0 +1,16 @@
+package infra
+
+import (
+	"fmt"
+	"os"
+)
+
+const UsageError = "Usage: stupid config.yaml 500\n"
+const ConnectError = "Error when connect with Fabric Network"
+const ConfigError = "Error Hapanned when Loading config.yaml"
+const CryptoError = "Error Hapanned with local MSP/Priv key"
```

Comment:
design as report error by types of connection, configuration, usage, and crypto settings.